frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/storage-tab/storage-tab.tsx

```diff
@@ -62,12 +62,14 @@ const getStoragesData = (
       pvcsLoading: !isLoaded(pvcs),
     });
 
+    const source = combinedDisk.getSource();
+
     return {
       wizardStorageData,
       // for sorting
       name: combinedDisk.getName(),
+      source: source && source.getValue(),
```

Comment:
maybe the safe accessing can move to `getSource()` ?

Reply:
fixed